test/accessibility/runPa11y.ts

```diff
@@ -24,7 +24,7 @@ const pa11yTest = pa11y({
 const accessibilityPages = [`${Paths.question}/121/62`];
 
 accessibilityPages.forEach(page => {
-  describe(`Running Accessibility tests for: ${page}`, () => {
+  describe.skip(`Running Accessibility tests for: ${page}`, () => {
```

Comment:
Please can you add a new story to sort these out so we don't forget?

Reply:
Ok